From 82091d9e2de2761daa36f8ef79f78a2cbd2a9f82 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Wed, 13 Jan 2016 11:56:15 +0800
Subject: [PATCH 1429/1691] MLK-12248-2 ARM: imx6sl-evk: add tx-d-cal property
 for usbphy

commit 82091d9e2de2761daa36f8ef79f78a2cbd2a9f82 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

Set tx-d-cal to be 0x5 to improve usb signal quality.

Signed-off-by: Li Jun <jun.li@nxp.com>
---
 arch/arm/boot/dts/imx6sl-evk.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sl-evk.dts b/arch/arm/boot/dts/imx6sl-evk.dts
index ec52ab1..e017696 100644
--- a/arch/arm/boot/dts/imx6sl-evk.dts
+++ b/arch/arm/boot/dts/imx6sl-evk.dts
@@ -916,6 +916,14 @@
 	status = "okay";
 };
 
+&usbphy1 {
+	tx-d-cal = <0x5>;
+};
+
+&usbphy2 {
+	tx-d-cal = <0x5>;
+};
+
 &usdhc1 {
 	pinctrl-names = "default", "state_100mhz", "state_200mhz";
 	pinctrl-0 = <&pinctrl_usdhc1>;
-- 
1.9.1

